lib/src/user.dart

```diff
@@ -77,6 +77,13 @@ class OneSignalUser {
     return await _channel.invokeMethod("OneSignal#removeTags", tags);
   }
 
+  /// Returns the list of tags on the current user.
```

Comment:
nit: let's also mention it is **"local tags"** to indicate this will not result in a server call, and could be out of date if they are also calling the REST API